pkg/universe.dagger.io/x/carvel.dev/test/test.bats

```diff
@@ -0,0 +1,11 @@
+setup() {
+    load '../../../bats_helpers'
+
+    common_setup
+}
+
+@test "kapp" {
+    dagger "do" -p ./test.cue test deploy
+    dagger "do" -p ./test.cue test verify
+    dagger "do" -p ./test.cue test delete
+}
```

Comment:
Since we do not deploy a kubernetes cluster in the CI, those test will never succeed.
@aluzzardi Should we deploy a kind cluster like we did pre europa?

Reply:
do let me know, what is needed here. Should I remove test.bats?

Reply:
I think you can comment it, that should be enough for now

Reply:
deleted the file for now. was unable to run it locally, so havent added it.